python/GangaLHCb/__init__.py

```diff
@@ -46,7 +46,8 @@
     configLHCb.addOption('SplitByFilesBackend', 'OfflineGangaDiracSplitter',
                      'Possible SplitByFiles backend algorithms to use to split jobs into subjobs,\
                       options are: GangaDiracSplitter, OfflineGangaDiracSplitter, splitInputDataBySize and splitInputData')
-
+    defaultLHCbDirac = 'v8r2p36'
+    configLHCb.addOption('diracVersion',defaultLHCbDirac,"set LHCbDirac version")
```

Comment:
Convention for naming config options is usually `UpperCamelCase` (but in some places `snake_case` is used).